manifests/base/runtimes/pretraining/mpi_distributed.yaml

```
@@ -12,36 +12,40 @@ spec:
       numProcPerNode: 1
       mpiImplementation: OpenMPI
       sshAuthMountPath: /home/mpiuser/.ssh
+      runLauncherAsNode: true
   template:
     spec:
       network:
-        publishNotReadyAddresses: false
+        publishNotReadyAddresses: true
       successPolicy:
         operator: All
         targetReplicatedJobs:
           - launcher
       replicatedJobs:
         - name: launcher
           template:
+            metadata:
+              labels:
+                trainer.kubeflow.org/trainjob-ancestor-step: trainer
             spec:
               template:
                 spec:
                   containers:
-                    - name: launcher
-                      image: mpioperator/mpi-pi:openmpi
-                      securityContext:
-                        runAsUser: 1000
-                      command:
-                        - mpirun
-                      args:
-                        - /home/mpiuser/pi
-        - name: trainer-node
+                  - name: launcher
```

Since we run Launcher as Node can we keep the container name: `node` ?
```suggestion
                  - name: node
```